cmd/ethrex/initializers.rs

```diff
@@ -183,6 +183,13 @@ pub async fn init_network(
     tracker: TaskTracker,
     blockchain: Arc<Blockchain>,
 ) {
+    let p2p_enabled = matches.get_flag("p2p.enabled");
+
+    if !p2p_enabled {
+        info!("P2P is disabled");
+        return;
+    }
```

Comment:
Should we put this before calling the function instead of inside it?

Reply:
Yeah, I will

Reply:
Done [46ed389](https://github.com/lambdaclass/ethrex/pull/2230/commits/46ed389d87a5561d665922be2318f310d05a84ac)